doc/DataSlice.md

```diff
@@ -43,13 +43,13 @@ Below is the metadata list and a note explaining how they are calculated.
     execution_end = timer()
     ```
 
-    * `conversion_time`
+    * `extraction_time`
 
     ```
-    convert_start = timer()
+    extract_start = timer()
     ....
     values = pd.DataFrame.from_dict(query_res.fetch*())
     ....
-    convert_end = timer()
+    extract_end = timer()
     ```
 
```

Comment:
Instead of having psuedo code related to snowflake, maybe we could have a quick diagram to present the steps and their measure? Or just a list like:
```
Steps what happens in get_slice:
- A: modify query to get just the slice
- B: request the result of the modified query to the external service 
- C: turn them into a DataFrame
- D: filter them with permissions
- E: compute statistics and return

`execution time`: start of A -> end of D
`extraction time`: start of B -> end of B
`conversion time`: start of C -> end of C
```

What do you think of this way of explaining?

Reply:
Listing all the steps involved will also help structuring the timings we need to measure, and how to name them :)